taichi/transforms/simplify.cpp

```diff
@@ -1,9 +1,132 @@
 #include <set>
 #include <unordered_set>
+#include <utility>
 #include "taichi/ir/ir.h"
 
 TLANG_NAMESPACE_BEGIN
 
+// Find if there is a load following a store in a basic block
```

Comment:
It seems to me that `LocalLoadSearcher` is just finding loads, since `LocalStore` is not mentioned here. I'm confused...

Reply:
Oh yes, `LocalLoadSearcher` is finding loads all over the `root`, with `root` actually be a statement after `LocalStore`, passed in as an argument. I'd better change the comment to just `Find if there is a load`.